@here/harp-mapview/lib/MapView.ts

```diff
@@ -2604,6 +2604,8 @@ export class MapView extends THREE.EventDispatcher {
             this.m_targetWorldPos.copy(target);
             this.m_targetGeoPos = this.projection.unprojectPoint(target);
             this.m_targetDistance = this.camera.position.distanceTo(target);
+        } else {
+            console.warn("Target not found");
```

Comment:
When could this happen?

Reply:
This is just for debug purposes... Should be removed in the final commit, I was checking with this if everything is fine with the target calculation code.